gantt_view.php

```diff
@@ -23,7 +23,7 @@
 
 
 
-// Verify the report exists, is a Gantt report, and is visible.
+// Verify the report exists and is a Gantt report.
```

Comment:
This updated comment reflects more accurately what is happening below, but should we not also check the visible flag is set, as indicated in the previous version of the comment?

Reply:
If a report is set as not visible it is hidden from the report list, but this is not an access control. Reports which are not visible may still be used e.g. as embedded images, saved to file fields, API endpoints or public reports. While most of those don't apply to Gantt charts, it may still be desirable to hide it in the list and provide a link to it elsewhere.